### PR TITLE
Add Bega Translation, add string to Madara "Ongoing"

### LINKED ISSUE #5007: Añadir Bega Translation
### Source name

Bega Translation

### Source link

https://begatranslation.com/

### Source language

Español

### Other details

_No response_

### Acknowledgements

- [X] I have checked that the extension does not already exist by searching the [extension listing](https://keiyoushi.github.io/extensions/) and verified it does not appear there.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a meaningful title with the source name.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by Smol-Ame**
Closes #5007 

* Add Bega Translation (ES)
 * I think it uses newChapterEndpoint, please advise if incorrect
 * Also, any easy way to nab mangaDetails cover to replace slightly low-res Popular/Browse?
* Bumped Madara theme
 * Added "Publicándose" to Madara theme Ongoing strings
 * If no bump like before, pls advise

Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [ ] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
- [x] Have removed `web_hi_res_512.png` when adding a new extension

### CODE REVIEW

File: src/es/begatranslation/src/eu/kanade/tachiyomi/extension/es/begatranslation/BegaTranslation.kt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import java.util.Locale
+
+class BegaTranslation : Madara(
+    "Bega Translation",
+    "https://begatranslation.com",
+    "es",
+    SimpleDateFormat("dd/MM/yyyy", Locale.ROOT),
+) {
+    override val useNewChapterEndpoint = true
+    override val mangaSubString = "mangas"
```
Comment (AwkwardPeak7) [inline src/es/begatranslation/src/eu/kanade/tachiyomi/extension/es/begatranslation/BegaTranslation.kt]:
```suggestion
    override val mangaSubString = "series"
```
generally mangasubeting is from the manga path => `https://begatranslation.com/series/cool-manga/`

File: lib-multisrc/madara/build.gradle.kts [hunk 2]
```diff
@@ -2,7 +2,7 @@ plugins {
     id("lib-multisrc")
 }
 
-baseVersionCode = 36
+baseVersionCode = 37
```
Comment (AwkwardPeak7) [inline lib-multisrc/madara/build.gradle.kts]: you can avoid bumping madara for this

**Review by AwkwardPeak7 — CHANGES_REQUESTED** [review]

**Comment by AwkwardPeak7** [comment]
> Also, any easy way to nab mangaDetails cover to replace slightly low-res Popular/Browse?

you can try writing an interceptor to remove `widthxheight` at end of url; or override mangadetailsparse to do the modifications there

**Comment by vetleledaal** [comment]
> * I think it uses newChapterEndpoint, please advise if incorrect

If you have set it to true it won't even try the old method, so it working means it's fine (or the chapters were found in the HTML).

> * Also, any easy way to nab mangaDetails cover to replace slightly low-res Popular/Browse?

override `popularMangaFromElement`, call the super method, and replace width/height. Something like:

https://github.com/keiyoushi/extensions-source/blob/6e79ac8a9abbbff223b82112365f9d65b32fe5cb/src/ja/mangarawplus/src/eu/kanade/tachiyomi/extension/ja/mangarawplus/MangaRawPlus.kt#L21-L25

**Review by vetleledaal — APPROVED** [review]

**Review by AwkwardPeak7 — APPROVED** [review]

**Pull request merged**